### DISCUSSION TITLE
Editing The Home Page

### DISCUSSION THREAD

**Discussion by trophyguy** [mcguirepr89/BirdNET-Pi#808] [General]
I'd like to do a couple edits on the home page. Instead of the generic BirdNET-Pi above the menubar
i'd like to add my site's name there. When I go into File Manager and edit index.php upon saving the file
I get the error _**Could Not Write File!! Check Ownership.**_

Do I have to change the permissions from 0664 or have the program shut down?

Also, seems it takes around 10-15 seconds sometime for the views.php to display on my homepage.
something i've mis-configured along the way? [My Birdnet](http://birdnet.cnyweather.com)

I love this overall. So many birds i've not know to be in my yard. 107 species so far in less than a year!!!!
Thank you!!

**Comment by ehpersonal38** [comment]
Hi, there was a slight bug in the permissions, I've fixed that now in the repo. You can run this command to fix it on your system:
`find $HOME/BirdNET-Pi -name '*.php' -exec chmod a+w {} \;`

Also I checked out your site and it's indeed behaving a little weirdly. I don't really know what's wrong there. My best advice to you would probably just be to reimage your system. 🙁

**Reply by trophyguy** [reply]
Reimage......would I lose all my data from when I started?

**Comment by ehpersonal38** [comment]
Just backup the following files and directories and you'll be able to
restore them after the reimage:
~/BirdNET-Pi/birdnet.conf,
~/BirdNET-Pi/apprise.txt,
~/BirdNET-Pi/scripts/birds.db,
~/BirdSongs/Extracted/By_Date
~/BirdSongs/Extracted/Charts
